Bottom: 2d1fce389a548a7bc23ff21150b6d283845b44da
Top:    036ef9d2dac828d63d5e5618d66e7f6bd7d3aae1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 21:38:21 +0100

hid/gtk: Invalidate the crosshair rather than drawing it immediately


---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 0df478d..6cb6e56 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1893,9 +1893,9 @@ PanAction (int argc, char **argv, int x, int y)
     }
   else if (x == on_x && y == on_y)
     {
-      ghid_show_crosshair (FALSE);
+      notify_crosshair_change (false);
       ghidgui->auto_pan_on = !ghidgui->auto_pan_on;
-      ghid_show_crosshair (TRUE);
+      notify_crosshair_change (true);
     }
 
   return 0;
